src/Components/Sliders/Popular/SliderPopular.scss

```diff
@@ -49,29 +43,31 @@
     color: green;
 }
 
-.slider__buttons {
+.slider-popular__buttons {
     display: inline-block;
     margin-left: 32px;
 }
 
-.slider__buttons button {
+.slider-popular__buttons button {
     padding: 10px 19px;
     color: green;
     border: 1px solid green;
     border-radius: 3px;
     background-color: white;
 }
 
-.slider__container {
+.slider-popular__slider {
     display: block;
     width: 1300px;
 }
 
-.slider__col {
+.slider-popular__col {
     max-width: 197px;
     height: 331px;
 }
 
-.slider__body {
-    width: 1300px;
+@media (max-width: 1100px) {
+    .slider-popular__col {
+        width: 224px;
```

Comment:
Slider SHOULD set width by himself. You do not need to worry about it. Just try to set padding: 0 xxx;

Reply:
Done.